### PR TITLE
now installing libaec-dev in developer workflow

### LINKED ISSUE #467: AEC code not appearing in code coverage report
Maybe because of the way that it is being handled in CMake.

@EricEngle-NOAA this is why we can't see your code coverage...

### PR DESCRIPTION
**Pull request by edwardhartnett**
Fixes #467

### CODE REVIEW

File: .github/workflows/developer.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
     - name: install-dependencies
       run: |
         sudo apt-get update
-        sudo apt-get install libaec-dev libpng-dev zlib1g-dev libjpeg-dev doxygen libopenjp2-7-dev valgrind
+        sudo apt-get install libaec-dev libpng-dev zlib1g-dev libjpeg-dev doxygen libopenjp2-7-dev valgrind libaec-dev
```
Comment (EricEngle-NOAA) [inline .github/workflows/developer.yml]: libaec-dev is listed first and last here.

**Review by EricEngle-NOAA — CHANGES_REQUESTED** [review]
See comment for apt-get install command.

File: .github/workflows/developer.yml (hunk 1 above)
Comment (edwardhartnett) [reply .github/workflows/developer.yml]: Duh! I will fix.

**Review by EricEngle-NOAA — APPROVED** [review]

**Comment by edwardhartnett** [comment]
@AlexanderRichert-NOAA or @Hang-Lei-NOAA could you please review?

**Review by Hang-Lei-NOAA — APPROVED** [review]

**Pull request merged**